includes/gateway/class-omise-payment-paynow.php

```diff
@@ -0,0 +1,153 @@
+<?php
```

Comment:
Looking at a lot of these `class-omise-payment-xxxxx.php` files, there is a lot of replication. Should we consider a hierarchy of classes here - to prevent further duplication and simplify existing classes?

Reply:
I could also see some part of the code that could be lean a bit more (`__construct`, `charge`, and `result` methods). But as for business perspective, probably not so efficient if we to do it at this time (as refactoring will take a lot of time to discuss and review).

But noted and acknowledged that we could improve those `class-omise-payment-xxxxx.php` classes a bit more.
I'll look into it after #149, #152 and one more coming PR, "API localization" get merged and released.
Thanks 👍